internal/command/postgres/import.go

```diff
@@ -82,6 +84,13 @@ func runImport(ctx context.Context) error {
 		imageRef  = flag.GetString(ctx, "image")
 	)
 
+	flapsClient, err := flapsutil.NewClientWithOptions(ctx, flaps.NewClientOpts{
+		AppName: appName,
+	})
+	if err != nil {
+		return fmt.Errorf("list of machines could not be retrieved: %w", err)
+	}
```

Comment:
Since we fetch the `AppCompact` below, you could move this down and put the `AppCompact` in the [`NewClientOpts`](https://github.com/superfly/fly-go/blob/cc398257f0c97d400e9438ac9b5c7f993261bc73/flaps/flaps.go#L42-L43). This would save a round-trip to the API … if the user is connecting to the Machines API over WireGuard, so this is tiny and probably not important. But you can if you want!

Reply:
that's great!